folium/map.py

```diff
@@ -60,6 +60,20 @@
      'https://rawgit.com/python-visualization/folium/master/folium/templates/leaflet.awesome.rotate.css'),  # noqa
     ]
 
+def _validate_location(values):
```

Comment:
Just this one left to address. I'll merge as is and fix it later. Thanks @radumas!